docs/02-app/01-building-your-application/02-data-fetching/01-fetching-caching-and-revalidating.mdx

```diff
@@ -211,7 +211,7 @@ You can also use the experimental [`unstable_cache` API](/docs/app/api-reference
 In the example below:
 
 - The React `cache` function is used to [memoize](/docs/app/building-your-application/caching#request-memoization) data requests.
-- The `revalidate` option is set to `3600` in the `layout.ts` and `page.ts` segments, meaning the data will be cached and revalidated at most every hour.
+- The `revalidate` option is set to `3600` in the `layout.tsx` and `page.tsx` segments, meaning the data will be cached and revalidated at most every hour.
```

Comment:
Since there is a JS / TS code switcher below, what about using the terms here instead of explicit file names?

```suggestion
- The `revalidate` option is set to `3600` in the Layout and Page segments, meaning the data will be cached and revalidated at most every hour.
```

Reply:
Good suggestion @manovotny 🙏🏼